File: 01-Assignment-02-02-2026/Using_C/program_23.c
/* Program 23: Count odd numbers in an array */

int isOdd(int num){
    if (num%2 == 0){
        return 0;
    }
    return 1;
}

#include <stdio.h>

int main() {
    int n;
    printf("Enter Length of Array : ");
    scanf("%d",&n);

    int array[n];
    for(int i=0;i<n;i++){
        printf("Enter No.%d : ",i+1);
        scanf("%d",&array[i]);
    }
    int count = 0;
    for(int i=0;i<n;i++){
        if (isOdd(array[i])){
            count++;
        }
    }
    printf("Total No. of Odd Number in array are %d",count);
    return 0;
}

File: 01-Assignment-02-02-2026/Using_Java/Program23.java
/*
Program 23
Java program to find the number of odd numbers in an array
*/

import java.util.Arrays;
import java.util.Scanner;

public class Program23 {
    public static boolean isOdd(double n) {
        if (n % 2 != 0)
            return true;
        else
            return false;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the size of array : ");
        int size = scanner.nextInt();

        double[] array = new double[size];

        System.out.println("Lets Fill the Array : ");
        for (int i = 0; i < size; i++) {
            System.out.print("Enter No." + (i + 1) + " : ");
            array[i] = scanner.nextDouble();
        }
        scanner.close();
        System.out.println("Now let's Count,How many are odd ! " + Arrays.toString(array));
        int count = 0;
        for (int i = 0; i < size; i++) {
            if (isOdd(array[i]))
                count++;
        }
        System.out.println("So Total Odd Number in Your Array is -> " + count);
    }

}
